### PR TITLE
huangting

### PR DESCRIPTION
**Pull request by oliikuto**
Hand on work.

### CODE REVIEW

File: lesson-1/huangtings-task/lesson-task.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    """This class retain one kind of nuts which can be removed to eat
+    in an individually way,and will raise an error
+    if you didn't choose the right nuts.
+    """
+
+    def __init__(self, items={}):
+        """
+        through the initial dictionary of items
+        """
+        if type(items) != type({}):
```
Comment (xcodebuild) [inline lesson-1/huangtings-task/lesson-task.py:13]: 像这里用`type(item) != dict`就可以了

**Comment by xcodebuild** [comment]
话说这个`py`文件是干什么的，为什么`__main__`都没有，是一个库么

**Comment by oliikuto** [comment]
只是一个类，模拟一个eatNuts的过程

File: lesson-1/huangtings-task/lesson-task.py (hunk 1 above)
Comment (oliikuto) [inline lesson-1/huangtings-task/lesson-task.py:13]: Thanks，I have changed it again.发送自 --------- Original Message --------- From: She Jinxin To: Jnrainerds/lesson-task Date: Fri Nov 07 23:38:58 GMT+08:00 2014 Subject: Re: [lesson-task] huangting (#14) In lesson-1/huangtings-task/lesson-task.py:> @@ -0,0 +1,95 @@ > +import sys > + > +class eatNuts: > + """This class retain one kind of nuts which can be removed to eat > + in an individually way,and will raise an error > + if you didn't choose the right nuts. > + """ > + > + def **init**(self, items={}): > + """ > + through the initial dictionary of items > + """ > + if type(items) != type({}): 像这里用type(item) != dict就可以了—Reply to this email directly or view it on GitHub.

**Pull request merged**